src/containers/share-sound-to-tiktok-modal/ShareSoundToTikTokModal.tsx

```diff
@@ -42,6 +47,9 @@ const fileRequirementErrorMessages = {
 }
 
 const ShareSoundToTikTokModal = () => {
+  const mobile = isMobile()
```

Comment:
this works, we also have a useIsMobile() hook that does the same thing. i don't really have a strong preference, but probably we should pick a pattern 🤷